pkg/features/kube_features.go

```diff
@@ -980,7 +981,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DisableNodeKubeProxyVersion: {Default: false, PreRelease: featuregate.Alpha},
 
-	DevicePluginCDIDevices: {Default: true, PreRelease: featuregate.Beta},
+	DevicePluginCDIDevices: {Default: true, PreRelease: featuregate.GA},
```

Comment:
@klueska @bart0sh  see lines 972

this should be `DevicePluginCDIDevices: {Default: true, PreRelease: featuregate.GA,  LockToDefault: true}, // remove in 1.33

Reply:
@aojea Thank you for the info! I thought that locking should happen *after* graduation to GA. Is there any documentation on this regard?

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md?plain=1 

 should we allow users to disable this GA feature?

Reply:
Thanks, will lock it in a separate PR.

Reply:
done: https://github.com/kubernetes/kubernetes/pull/125687